.github/dependabot.yml

```diff
@@ -23,3 +33,20 @@ updates:
       - dependency-name: "jmespath"
       - dependency-name: "urllib3"
       - dependency-name: "wheel"
+
+  - package-ecosystem: "pip"
+    directory: "/"
+    open-pull-requests-limit: 10
+    schedule:
+      interval: "weekly"
+    target-branch: "develop"
+    labels:
+      - "dependencies"
+      - "v1"
+    allow:
+      - dependency-name: "colorama"
+      - dependency-name: "docutils"
+      - dependency-name: "awscrt"
```

Comment:
This actually probably doesn't need to be managed here. It would be pulled in through botocore.
```suggestion
```